style.css

```diff
@@ -72,11 +78,15 @@ h1 {
 
 #socials {
   display: flex;
-  justify-content: space-around;
   align-items: center;
-  padding: 25%;
+  background-repeat: no-repeat;
 }
 
 .mediumlogo {
-  margin-top: 0.5vh;
+  margin-top: 0.8vh;
+}
+
+.logos {
```

Comment:
- [ ] It would be better if we use the recommended color for our social media logos, here is the recommended color `#505F79`

Reply:
That's an actual wrong css mistake by me there is as a matter of fact no need for a color at all i was checking if could make the background was not transparent but then i realised it was just that items were not behind a box but instead not in the same position. If I apply any color the "logos" box will not match the given template. So sorry for this